### PR TITLE
 Fix exception with reading non-string metadata keys in config.

### PR DESCRIPTION
**Pull request by inferno-chromium**

### CODE REVIEW

**Comment by inferno-chromium** [comment]
This is fix for exception after my last commit, since device-memory-mb is int (4096)

```
AttributeError: 'int' object has no attribute 'startswith'
at _process_instance_template (/base/data/home/apps/s~cluster-fuzz/cron-service:20190325t170232.417023464658087344/python/google_cloud_utils/compute_engine_projects.py:61)
at _config_to_project (/base/data/home/apps/s~cluster-fuzz/cron-service:20190325t170232.417023464658087344/python/google_cloud_utils/compute_engine_projects.py:84)
at load_project (/base/data/home/apps/s~cluster-fuzz/cron-service:20190325t170232.417023464658087344/python/google_cloud_utils/compute_engine_projects.py:115)
at __init__ (/base/data/home/apps/s~cluster-fuzz/cron-service:20190325t170232.417023464658087344/handlers/cron/manage_vms.py:164)
at get (/base/data/home/apps/s~cluster-fuzz/cron-service:20190325t170232.417023464658087344/handlers/cron/manage_vms.py:706)
at wrapper (/base/data/home/apps/s~cluster-fuzz/cron-service:20190325t170232.417023464658087344/libs/handler.py:97)
at dispatch (/base/alloc/tmpfs/dynamic_runtimes/python27g/931d17f05408b3ef/python27/python27_lib/versions/third_party/webapp2-2.3/webapp2.py:545)
```

File: src/python/google_cloud_utils/compute_engine_projects.py [hunk 1]
```diff
@@ -58,7 +58,8 @@ def _process_instance_template(instance_template):
   # Load file metadata
   items = instance_template['properties']['metadata']['items']
   for item in items:
-    if item['value'].startswith(FILE_SCHEME):
+    if (isinstance(item['value'], basestring) and
```
Comment (mbarbella-chromium) [inline src/python/google_cloud_utils/compute_engine_projects.py]: Could you add a comment explaining this here or a potential FIXME to clean this function up a bit more? This seems fine for an exception fix but the function is a bit confusing.

Comment (inferno-chromium) [reply src/python/google_cloud_utils/compute_engine_projects.py]: Done. Added more detailed comments to explain purpose, but a FIXME is not required.

**Pull request merged**